app/services/rummager_link_sync.rb

```diff
@@ -0,0 +1,27 @@
+class RummagerLinkSync
```

Comment:
`RummagerLinkSync` does not make clear to me what this thing is. Maybe `RummagerLinkSynchronizer`?